CHANGELOG.md

```diff
@@ -8,10 +8,16 @@ This is the log of notable changes to the Expo client that are developer-facing.
 
 ### 🛠 Breaking changes
 
+- `expo-facebook` All methods and platforms now return time intervals in JS `Date` objects instead of seconds. ([#7101](https://github.com/expo/expo/pull/7101) by [@evanbacon](https://github.com/evanbacon))
```

Comment:
The affected values are all times, not time intervals. I don't think we ever (?) want to use Date objects for intervals (there might be exceptions but can't think of why right now).

```suggestion
- `expo-facebook` All methods and platforms now return dates as JS `Date` objects instead of seconds. ([#7101](https://github.com/expo/expo/pull/7101) by [@evanbacon](https://github.com/evanbacon))
```